infrastructure/template.json

```diff
@@ -0,0 +1,183 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serverfarms_ASP_test_bfbf_name": {
+            "defaultValue": "ASP-test-bfbf",
+            "type": "String"
+        },
+        "sites_TechnicalDocuIndexerWeb_name": {
+            "defaultValue": "TechnicalDocuIndexerWeb",
+            "type": "String"
+        },
+        "components_TechnicalDocuIndexerWeb_name": {
+            "defaultValue": "TechnicalDocuIndexerWeb",
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "microsoft.insights/components",
+            "apiVersion": "2018-05-01-preview",
+            "name": "[parameters('components_TechnicalDocuIndexerWeb_name')]",
+            "location": "westeurope",
+            "kind": "web",
+            "properties": {
+                "Application_Type": "web",
+                "Flow_Type": "Redfield",
+                "Request_Source": "IbizaWebAppExtensionCreate",
+                "RetentionInDays": 90,
+                "publicNetworkAccessForIngestion": "Enabled",
+                "publicNetworkAccessForQuery": "Enabled"
+            }
+        },
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "2018-02-01",
+            "name": "[parameters('serverfarms_ASP_test_bfbf_name')]",
+            "location": "West Europe",
+            "sku": {
```

Comment:
I would make SKU as a parameter 😃

Reply:
I understand the need, but we will not make it more than the Free tier, as to not waste $$.
Nonetheless in the normal project, I would make it as parameters